server/controllers/AuthController.js

```diff
@@ -112,4 +111,56 @@ export default class AuthController {
       return responseMessage(response, 500, { message: error.message });
     }
   }
+
+/**
+ *
+ *
+ * @static
+ * @param {*} request
+ * @param {*} response
+ * @returns {object} - response object
+ * @memberof AuthController
```

Comment:
params should be object and returns json

Reply:
noted